src/esp/bindings/AttributesBindings.cpp

```diff
@@ -164,12 +164,11 @@ void initAttributesBindings(py::module& m) {
       .def_property(
           "shader_type", &AbstractObjectAttributes::getShaderType,
           &AbstractObjectAttributes::setShaderType,
-          R"(The shader type [0=flat, 1=phong, 2=pbr] to use for this construction)")
+          R"(The shader type [0=material, 1=flat, 2=phong, 3=pbr] to use for this construction)")
```

Comment:
Wait is this an enum? If not, it really should be.

Reply:
It is.  I just need to bind it.  That's coming next, I just need to get this fix to the stage lighting in asap.